### ISSUE TITLE
Boundaries debug mode

### ISSUE THREAD

**Issue by crashmax-dev** [crashmax-dev/fireworks-js#23]
- The border does not work correctly, at certain border sizes, the fireworks fly to the left vertical.
![fireworks-boundaries](https://user-images.githubusercontent.com/15673111/135702789-7dea276b-c0b1-47df-a849-00ff0b46fceb.gif)

**Comment by mixaildudin** [comment]
Hey!

Took a quick look at the code. Seems like it's this logic that causes the problem:
` randomInt(this.boundaries.x, this.boundaries.width - this.boundaries.x * 2)`

Say, we have width = 200, x = 100. This code will generate a random int between 100 and 200 - 100 * 2 = 0. Which is just the same as having `randomInt(0, 200)`.

So this problem will occur whenever `width - 2x < x <=> width < 3x`. When I set x = 100 and width = 300, it goes well.

At first I thought the boundaries were doing strange things: I increase `x` in the demo, so the rectangle is supposed to move right, isn't it? But instead, it stands still but shrinks. I wanted to submit a PR and make the boundaries behave more naturally, but it looks like you intentionally programmed this behaviour.

So I decided that maybe this comment would help and save you some time :)
